department-of-veterans-affairs/caseflow
# Team Queue and Individual Correspondence Page - Framework 35471

Resolves [Team Queue and Individual Correspondence Page - Framework](https://jira.devops.va.gov/browse/APPEALS-35471)

# Description
This change converts the QueueTable in the correspondence page to a QueueTableBuilder. It also does so for the Correspondence team, and will execute queries on the backend before handing the information to the frontend.

## Acceptance Criteria

## Testing Plan


1. Visit queue/correspondence as jon snow
2. manipulate the pagination so that the page changes. Notice that the URL changes with the filters applied.
3. visit the team page for correspondence
4. see that it's also filtering on the backend.

# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="1297" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/c08756a7-83af-4fff-a6cc-1ac2dcb9d42c">|<img width="1431" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/f7dfcb49-5e5c-46cd-af7b-5da19da8d2bb">

### Code Climate
Your code does not add any new code climate offenses? If so why?
